source/templates.tex

```diff
@@ -1483,7 +1483,7 @@
 the
 \grammarterm{template-argument}
 for
-\tcode{Array<T>\,::\,op\-er\-a\-tor\,[]\,()}
+\tcode{Array<T>::op\-er\-a\-tor\,[]\,()}
```

Comment:
Unrelatedly, this line has an instance of a function being referred to as "f()" even though it doesn't have zero parameters and cannot be called with zero arguments. That practice seems a bit sloppy for the standard (and certainly isn't used consistently in the document). Thoughts?